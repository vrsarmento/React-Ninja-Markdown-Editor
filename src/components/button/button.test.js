'use strict'

import React from 'react'
import Button from './index'
import renderer from 'react-test-renderer'

it('Should Button Default to match snapshot', () => {
  const tree = renderer
    .create(<Button onClick={() => null}>Click me</Button>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Should Button Success to match snapshot', () => {
  const tree = renderer
    .create(<Button onClick={() => null} kind='success'>Click me</Button>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Should Button Info to match snapshot', () => {
  const tree = renderer
    .create(<Button onClick={() => null} kind='info'>Click me</Button>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('Should Button Danger to match snapshot', () => {
  const tree = renderer
    .create(<Button onClick={() => null} kind='danger'>Click me</Button>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
