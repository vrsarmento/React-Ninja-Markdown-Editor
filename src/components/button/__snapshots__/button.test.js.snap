// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Button Danger to match snapshot 1`] = `
<button
  className="button -danger"
  onClick={[Function]}
>
  Click me
</button>
`;

exports[`Should Button Default to match snapshot 1`] = `
<button
  className="button"
  onClick={[Function]}
>
  Click me
</button>
`;

exports[`Should Button Info to match snapshot 1`] = `
<button
  className="button -info"
  onClick={[Function]}
>
  Click me
</button>
`;

exports[`Should Button Success to match snapshot 1`] = `
<button
  className="button -success"
  onClick={[Function]}
>
  Click me
</button>
`;
